shared/src/business/useCaseHelper/trialSessions/setNoticeOfChangeToRemoteProceeding.js

```diff
@@ -0,0 +1,146 @@
+const {
+  aggregatePartiesForService,
+} = require('../../utilities/aggregatePartiesForService');
+const {
+  CASE_STATUS_TYPES,
+  DOCUMENT_PROCESSING_STATUS_OPTIONS,
+  SYSTEM_GENERATED_DOCUMENT_TYPES,
+  TRIAL_SESSION_PROCEEDING_TYPES,
+} = require('../../entities/EntityConstants');
+const { DocketEntry } = require('../../entities/DocketEntry');
+
+const serveNoticesForCase = async (
+  applicationContext,
+  {
+    caseEntity,
+    newPdfDoc,
+    noticeDocketEntryEntity,
+    noticeDocumentPdfData,
+    PDFDocument,
+    servedParties,
+  },
+) => {
+  await applicationContext.getUseCaseHelpers().sendServedPartiesEmails({
+    applicationContext,
+    caseEntity,
+    docketEntryId: noticeDocketEntryEntity.docketEntryId,
+    servedParties,
+  });
+
+  if (servedParties.paper.length > 0) {
+    const noticeDocumentPdf = await PDFDocument.load(noticeDocumentPdfData);
+
+    await applicationContext
+      .getUseCaseHelpers()
+      .appendPaperServiceAddressPageToPdf({
+        applicationContext,
+        caseEntity,
+        newPdfDoc,
+        noticeDoc: noticeDocumentPdf,
+        servedParties,
+      });
+  }
+};
+
+/**
+ * setNoticeOfChangeToRemoteProceeding
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {object} providers.caseEntity the case data
```

Comment:
Nit: a couple args are missing in these docs

Reply:
Thanks! TODO: add docs